### PR TITLE
Download latest released models

### PR DESCRIPTION
**Pull request by ar13pit**

### CODE REVIEW

**Comment by ar13pit** [comment]
@MatthijsBurgh I can't wait for tue-robotics/tue-env#377 to be merged before finishing up this system. So I have opened an issue to generalize the path dirs after the merge in tue-env.

**Comment by ar13pit** [comment]
@MatthijsBurgh review please

File: ros-speech_recognition/install.bash [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+url=$(curl -s https://api.github.com/repos/tue-robotics/speech_recognition/releases/latest | \
```
Comment (MatthijsBurgh) [inline ros-speech_recognition/install.bash]: Check ros-dashboard target, it uses some python to do this.

Comment (ar13pit) [reply ros-speech_recognition/install.bash]: It is pretty much the same. Here I am specifying the version to always be the latest one while there it takes a version as an argument and searches over all the versions.

What do you want me to do? That's an overkill for what I am doing here.

Comment (MatthijsBurgh) [reply ros-speech_recognition/install.bash]: True, but it uses json in python is cleaner IMO.

**Comment by ar13pit** [comment]
@MatthijsBurgh Change to python for parsing the github releases added as a issue #30 . For now lets merge this unless you feel there is an error somewhere in the script.

**Pull request merged**